doc/source/extended_documentation/psychrometric_calculations/psychrometric_calculations/_calculate_phase_change_level.rst

```diff
@@ -0,0 +1,66 @@
+**Method**
+
+The first call is to ``find_falling_level`` which finds the height
+at which the wet-bulb integral reaches the required threshold, this
+threshold defining the phase change level. The threshold value depends
+upon the phase change that has been requested.
+
+The wet-bulb integral data used for this process is only available
```

Comment:
I think this paragraph could be a bit clearer. I think you explain the two cases which might cause missing data thoroughly below, so maybe this just needs to say something like:
If we are unable to find the height of the phase level for a grid point from the wet-bulb integral the the value at that point is set to np.nan and will be filled in later.

Reply:
Done.